README.md

```diff
@@ -1 +1,10 @@
-# QnA
\ No newline at end of file
+# QnA
+QnA Jabar Digital Service adalah platform untuk membantu teman-teman <i>Engineer</i> mengajukan pertanyaan, belajar, dan berbagi pengetahuan teknis.
+
+## Cara Bertanya
+- Silahkan submit pertanyaannya di bagian Issues di project ini
+- Pastikan dulu tidak ada pertanyaan yang sama dengan yang lain, jika sama lebih baik vote pertanyaan yang sudah ada, jika tetep memaksa membuat pertanyaan baru, maka pertanyaannya akan di close.
+Semua orang bisa ikut menjawab, feels free untuk kontribusi.
```

Comment:
Usul perbaikan redaksi agar lebih halus, dari semula:
> Semua orang bisa ikut menjawab, feels free untuk kontribusi.

Menjadi:
> Semua orang bisa ikut menjawab, feels free untuk memberikan pendapat. Kita menghargai setiap perbedaan pendapat yang terjadi dalam diskusi.

Reply:
sip mas @yohang88 betul banget redaksinya lebih halus yang disarankan. sudah saya revisi redaksinya

Reply:
sedikit nambahin saran redaksi lagi,

> Siapapun dapat berpartisipasi untuk memberikan pendapat dan jawaban. Mohon sampaikan pendapat dan jawaban anda dengan sopan, tidak menyinggung ataupun menyerang pribadi, dan bersifat SARA.

Reply:
Kalau terkait rules/code of conduct menurut saya bisa dipisahkan. Karena ini masih tahap awal, mungkin belum perlu dibuat yang mendalam.